### PR TITLE
[1LP][RFR] Cloud Intel / Reports widgetastic conversion. Part 2

### PR DESCRIPTION
**Pull request by quarckster**
Purpose
=================
This PR is a part of set of Cloud Intel / Reports widgetastic conversion PRs. Cloud Intel / Report / Schedules have been converted to widgetastic.

{{pytest: -v -k "test_schedule_crud"}}
58z failed due https://bugzilla.redhat.com/show_bug.cgi?id=1477901

### CODE REVIEW

File: cfme/intelligence/reports/__init__.py [hunk 1]
```diff
@@ -1,24 +1,86 @@
...
+            self.navigation.currently_selected == ["Cloud Intel", "Reports"])
+
+    @property
+    def is_displayed(self):
+        return self.in_intel_reports and self.configuration.is_displayed
+
+    @View.nested
+    class saved_reports(Accordion):  # noqa
+        ACCORDION_NAME = "Saved Reports"
+
```
Comment (izapolsk) [inline cfme/intelligence/reports/__init__.py]: please remove unnecessary spaces

Comment (quarckster) [inline cfme/intelligence/reports/__init__.py]: After merging #5078 it will be rebased.

**Comment by quarckster** [comment]
@izapolsk PRT failed because of https://bugzilla.redhat.com/show_bug.cgi?id=1477901

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**